### ISSUE TITLE
"Cannot find calendar. Check instructions for set up."

### ISSUE THREAD

**Issue by WolfmanArtist** [Davepar/gcalendarsync#66]
I followed all these instructions and placed my calendar's ID in the string of code just as it was explained here. Whenever I select a cell and click either "Sync to Calendar" or "Update to Calendar" on the top ui bar of the spreadsheet. I still get this error pop-up "Cannot find calendar. Check instructions for set up." The script is running fine, too.

Also keep in mind that I am a newb and might not have common sense with appscript.

**Comment by Davepar** [comment]
Did you double check that your have the calendar ID correct? It should end in "@group.calendar.google.com".

**Comment by WolfmanArtist** [comment]
Yeah, it ends in that. I also get a typeError each time I select one of the macros as a result of this being imported. Maybe I missed a step?  I downloaded the sheet, copy/pasted script, replaced the calendarID with the calendarID that I have, and then imported the script under Tools>Macros>Import

**Comment by Davepar** [comment]
That might be the issue. This is a script, not a macro. You should be doing Tools -> Script Editor and pasting the code there. I'm not exactly sure what the difference is, but that might be it.
